matrixprofile/algorithms/streamingimpl.pyx

```diff
@@ -0,0 +1,222 @@
+# -*- coding: utf-8 -*-
+#cython: boundscheck=True, cdivision=True, wraparound=True
+import numpy as np
+from cython.view cimport array
+from libc.math cimport sqrt
+from matrixprofile.cycore import muinvn
+
+# These are here for convenience purposes for now
```

Comment:
Feel free to clean up code that isn't used and you don't see value in keeping. For the name of this file, will you please stick to the convention I set of prefixing with "cy". For example, "streamingimpl.pyx" -> "cystreaming.pyx". If this is core functions, maybe "cystreamingcore.pyx". We may also consider refactoring some of the codes into different modules for clarity.

Reply:
I'll check this part.